pkgpanda/actions.py

```diff
@@ -15,28 +15,28 @@
 """
 
 
-def activate_packages(install, repository, package_ids, no_systemd=False, noblock_systemd=True):
+def activate_packages(install, repository, package_ids, systemd=True, block_systemd=False):
     """Replace the active package set with package_ids.
 
     install: pkgpanda.Install
     repository: pkgpanda.Repository
     package_ids: sequence of package IDs to activate
-    no_systemd: don't start/stop systemd services (default: False)
-    noblock_systemd: if not no_systemd, don't block waiting for systemd services to come up (default: True)
+    systemd: start/stop systemd services (default: True)
+    block_systemd: if systemd, block waiting for systemd services to come up (default: False)
 
     """
     assert isinstance(package_ids, collections.Sequence)
     try:
         install.activate(repository.load_packages(package_ids))
-        _start_dcos_target(no_systemd, noblock_systemd)
+        _start_dcos_target(systemd, block_systemd)
     except ValidationError as ex:
         print("Validation Error: {0}".format(ex))
         sys.exit(1)
     except PackageError as ex:
         print("Package Error: {0}".format(ex))
 
 
-def swap_active_package(install, repository, package_id, no_systemd=False, noblock_systemd=True):
+def swap_active_package(install, repository, package_id, systemd=False, block_systemd=False):
```

Comment:
Looking at the code, we never use these defaults actually. Why not just drop them?